src/lib/routes/admin-api/project/variants.ts

```diff
@@ -127,6 +147,9 @@ export default class VariantsController extends Controller {
             handler: this.patchVariantsOnEnv,
             middleware: [
                 openApiService.validPath({
+                    summary:
+                        'Patch variants for a feature toggle under the provided environment',
+                    description: `Returns the variants for a feature in a specific environment.`,
```

Comment:
This seems like a copy-paste mistake? Also, what format is the patch in? Is this operation atomic?

```suggestion
                    description: `Apply a patch to the features environments in the specified environment. The patch must follow [some format]. `,
```

Reply:
```suggestion
                    description: `Apply a patch to the features environments in the specified environment. The patch object should conform to the [JSON-patch format (RFC 6902)](https://www.rfc-editor.org/rfc/rfc6902).`,
```

Reply:
Yes, also to be precise, the patch object should conform to an array of JSON-patch format...

Reply:
A great!
```suggestion
                    description: `Apply a list of patches to the features environments in the specified environment. The patch objects should conform to the [JSON-patch format (RFC 6902)](https://www.rfc-editor.org/rfc/rfc6902).`,
```